cmd/krew/cmd/search.go

```diff
@@ -118,6 +113,33 @@ Examples:
 	PreRunE: checkIndex,
 }
 
+func searchByNameAndDesc(keyword string, names, descs []string) []string {
```

Comment:
it feels weird that names and descs are separate arrays.
can you create an internal struct to carry this info perhaps?

Reply:
Updated.